### PR TITLE
Add simple value tests related to nil, boolean and int

### PR DESCRIPTION
**Pull request by SandaruJayawardana**
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [x] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/ballerina-spec-conformance-tests/src/test/resources/5-values-types-and-variables/int.balt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+Test-case: error
+Label: int
+Author: SandaruJayawardana
+
+public function main() {
+    int x = 9223372036854775808;
+    io:println(x); // @error <Integer '9223372036854775808' too large>
+}
```
Comment (MaryamZi) [inline tests/ballerina-spec-conformance-tests/src/test/resources/5-values-types-and-variables/int.balt:60]: There's potentially few more tests like
- `DecimalNumber` cannot start with `0` unless the value itself is `0`. e.g., `01` should result in an error.
- `HexIntLiteral` must have both the `HexIndicator` and `HexNumber`

But these could be parser tests too. Let's discuss during the review and decide whether or not we should be adding such tests here.

File: tests/ballerina-spec-conformance-tests/src/test/resources/5-values-types-and-variables/boolean.balt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Test-case: succeed
```
Comment (chiranSachintha) [inline tests/ballerina-spec-conformance-tests/src/test/resources/5-values-types-and-variables/boolean.balt:1]: @MaryamZi  Here `succeed` should be changed to `output` . Right?
